ServerCore/Pages/Resources/ToolsPartial.cshtml

```diff
@@ -1,49 +1,68 @@
+    
+    <h3>Instructions</h3>
+    <p>If a puzzle does have instructions, there may also be clues in them beyond just their literal meaning.  Look at every word or phrase, because it may have a double meaning! It could be pointing out a mechanic, an encryption, or even a piece of information out on the Internet that you need to find in order to solve the puzzle.</p>
+
+    <h3>Extraction</h3>
+    <p>The final answer for a puzzle is usually a single word or short phrase. Longer phrases that you extract are often just next steps in the puzzle or clues that you're on the right track. When in doubt, put what you have into the answer submission system! Often, the final answer after a longer phrase is just a piece of information on the internet instead of directly in the puzzle since it's harder to accidentally solve the puzzle that way, so don't be afraid to look things up, either!</p>
 
-    <h3><a href="http://web.archive.org/web/20171224203122/http://www.crosswordman.com/teadown.html">The Electronic Alveary (TEA)</a></h3>
-    <p>TEA (The Electronic Alveary) allows you to search word lists for words that match a pattern. TEA's pattern language supports simple template and anagram searches, but also has the flexibility to perform many more complicated queries.  Check the Quick Reference page on the help menu to see the kinds of things TEA can do for you. Typing ;STIN for example will product all anagrams of the letters STIN (ISN'T, NITS, TINS, SNIT).</p>
+    <h3>Types of Puzzles</h3>
+    <p>One of the best ways to learn how to solve puzzles is just to familiarize yourself with all the different kinds of puzzles there are. For example, Japan is famous for its many types of number puzzles.  You've likely heard of Sudoku, but there are tons more!  You can explore and play samples of many of them at sites like <a href="https://www.puzzazz.com/how-to-solve" target="_blank">Puzzazz</a>, <a href="https://www.nikoli.co.jp/en/puzzles/" target="_blank">Nikoli</a>, and <a href="https://www.chiark.greenend.org.uk/~sgtatham/puzzles/" target="_blank">Simon Tatham's</a>. As you can see, some puzzle types even go by multiple names, and there are often types of puzzles that you didn't even know had a name!  Those groups of pictures that represent a word? That's called a "Rebus".</p>
+    <p>Similarly, board and card game mechanics are often used in puzzles, and the solver is expected to know the mechanic by name. Chess moves, Poker hands, and more are all fair game, and this is especially true when the game itself has puzzle-like mechanics.  For example, Boggle works just as well online as it does with the physical cubes!</p>
 
-    <h3><a href="https://nutrimatic.org/">Nutrimatic</a></h3>
-    <p>Nutrimatic is another powerful anagram solver that's backed by Wikipedia's database, so modern words are likely to be found.</p>
+    <h3>Wikipedia</h3>
```

Comment:
The section doesn't really focus on wikipedia that much. Should this title be updated?